lib/iex/lib/iex/info.ex

```diff
@@ -1,15 +1,15 @@
 defprotocol IEx.Info do
   @fallback_to_any true
 
-  @spec info(term) :: [{atom, String.t()}]
+  @spec info(term()) :: [{info_name :: String.t(), info :: String.t()}]
```

Comment:
This is a change to a public contract. We need to make sure we still accept atoms everywhere (I would use String.Chars.t and call it a day).

Reply:
Oh boy you're right, didn't think about others using this protocol. Fixed!